spec/support/page_objects/providers/show.rb

```diff
@@ -8,6 +8,8 @@ class Show < PageObjects::Base
       element :add_a_user, "a", text: "Add a user"
       element :edit_this_provider, "a", text: "Edit this provider"
 
+      element :delete_a_user, ".govuk-button--warning", match: :first
```

Comment:
A bit lazy?

Reply:
capybara has an ambiguous match because there are two buttons - did you have something else in mind?

Reply:
Something like 

https://github.com/DFE-Digital/register-trainee-teachers/blob/main/spec/support/page_objects/providers/index.rb#L12